spark-jobs/src/test/resources/data/standardization_fixed_width_suite_expected_non_trimmed.txt

```diff
@@ -0,0 +1,12 @@
++---+---------------+---------+--------------------------------------------------------------------------+
+|id |first_name     |last_name|errCol                                                                    |
++---+---------------+---------+--------------------------------------------------------------------------+
+|0  |Georgina       | Richardt|[[stdCastError, E00000, Standardization Error - Type cast, id, [1  ], []]]|
+|0  |Seamus         |    Klaes|[[stdCastError, E00000, Standardization Error - Type cast, id, [2  ], []]]|
+|0  |Griff          |  Leaning|[[stdCastError, E00000, Standardization Error - Type cast, id, [3  ], []]]|
+|0  |Klarrisa       |    Corps|[[stdCastError, E00000, Standardization Error - Type cast, id, [4  ], []]]|
+|0  |Cornie         |Hewertson|[[stdCastError, E00000, Standardization Error - Type cast, id, [5  ], []]]|
+|0  |Maybeeeeeeeeeee|        a|[[stdCastError, E00000, Standardization Error - Type cast, id, [6  ], []]]|
```

Comment:
👀  Interesting that type casting failed here

Reply:
`"  10".toLong` will fail as well. Regardless of the whitespace placement.

Reply:
Is it the expected behavior or should we log an issue?

IMHO, the current behavior makes non-trimming mode useless if there are numeric fields in the file.

Reply:
> 
> 
> Is it the expected behavior or should we log an issue?
> 
> IMHO, the current behavior makes non-trimming mode useless if there are numeric fields in the file.

Can use pattern, particuarly if the number of digits is fixed. But I agree, the usability is limited.

Reply:
We all agree it is weird. But the scope of this PR was to make something work again as it was. We can log further issues down the road or when we revive the library